sdk/js/src/Request.ts

```diff
@@ -103,18 +56,15 @@ export const getConfigJson = async (
         queryParams.toString()
 
     try {
-        const res = await get(url)
-        return res.data
-    } catch (ex: any) {
-        const errorString = JSON.stringify(ex?.response?.data?.data?.errors)
+        const res = await getWithTimeout(url, requestConfig, 5000)
```

Comment:
Was 5 seconds the timeout before? or is this new?

Reply:
The timeout was 5 seconds before, it was just being set on the axios client https://github.com/DevCycleHQ/js-sdks/pull/739/files#diff-b985bf038e49b8ee77cff70485df18db4d9fd1d6ff6992dd90a1038f9b547393L8